src/main/java/com/market/openmarket/common/util/UserValidator.java

```diff
@@ -6,13 +6,15 @@
 import com.market.openmarket.domain.user.UserRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
 
 @Component
 @RequiredArgsConstructor
 public class UserValidator {
 
     private final UserRepository userRepository;
 
+    @Transactional(readOnly = true)
```

Comment:
JPA 더티 체킹 방지를 위해 transaction을 ReadOnly로 설정을 하신거 같습니다.
해당 비지니스 로직에 더티 체킹이 발생하는 구간은 어디일까요?

Reply:
```suggestion
     @Transactional
    public User createUser(UserSignUpRequestDto requestDto) {
        userValidator.validateDuplicate(requestDto);
```
UserServiceImpl.createUser() 에서 해당 메서드를 호출합니다. 상위 트랜잭션은 write lock이고 유저 생성 시 더티체킹이 발생하지 않지만 직관적, 명시적으로 readonly 설정을 하였습니다.

Reply:
validateDuplicate 이 메서드는 객체의 데이터를 변경하는 행위를 할 까요? 해당 메서드외에서 수정이 발생하는 경우 아닐까요?